staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5073,6 +5073,9 @@ type NodeStatus struct {
 	// Note: This field is declared as mergeable, but the merge key is not sufficiently
 	// unique, which can cause data corruption when it is merged. Callers should instead
 	// use a full-replacement patch. See https://pr.k8s.io/79391 for an example.
+	// Besides, it is possible its addresses get changed because the Node or kubelet
```

Comment:
is the API docs the right place for this?

Reply:
Andrew suggested here and I think it makes sense.
https://github.com/kubernetes/kubernetes/pull/108095#issuecomment-1247059240